internal/cmd/config.go

```diff
@@ -1296,13 +1297,16 @@ func (c *Config) newSourceState(ctx context.Context, options ...chezmoi.SourceSt
 		sourceDirAbsPath = c.SourceDirAbsPath.JoinString(string(bytes.TrimSpace(data)))
 	}
 
+	httpClient := newCacheHTTPClient(c.CacheDirAbsPath.Join(httpCacheDirRelPath).String())
```

Comment:
Let's store the `HTTPClient` in the config (`c`) so we can re-use it elsewhere.

Reply:
I agree, I think so too
At first I wanted to do this, but I was worried that you might not like to change the `config`